ui/src/main/resources/Meeting/SkinExtension.xml

```diff
@@ -133,6 +133,9 @@
 
 #durationEdit select{
    width:auto ;
+}
+.col-sm-12.col-xs-12 label {
```

Comment:
Also this kind of specificity for the selector is not wanted. Both are grid classes and they are also targeting different resolutions so there is no semantic reason to use them as selectors.